tests/pytest/_cli_test.py

```diff
@@ -89,7 +89,8 @@ def populate_argument_parser(
             return None
 
         def invoke_cli_app(self, parsed_cli_args: Namespace) -> ReturnCodeType:
-            raise PreCommitTerraformExit('sentinel')
+            msg = 'sentinel'
+            raise PreCommitTerraformExit(msg)
```

Comment:
Wait, was this automatic? This seems manual. Drop “autofixes” from the CI title. Just say that you're fixing linting violations. It doesn't really matter if it's semiautomatic or not.

Reply:
Its automatic
 EM101 - Exception must not use a string literal, assign to variable first
[`93898c9` (#863)](https://github.com/antonbabenko/pre-commit-terraform/pull/863/commits/93898c98853448dd2b83353fd1b8f3589fe77b08)

Reply:
In this case, we can probably just reuse another sentinel var:
```suggestion
            raise PreCommitTerraformExit(self.CLI_SUBCOMMAND_NAME)
```

But in a more generic case, I'd rename `msg` to something meaningful. Like `program_exit_msg`.